cargo-concordium/src/build.rs

```diff
@@ -314,6 +314,45 @@ pub fn build_contract_schema<A>(
     Ok(schema)
 }
 
+/// Create a new Concordium smart contract project from a template, or there
+/// are runtime exceptions that are not expected then this function returns
+/// Err(...).
+///
+/// Otherwise a boolean is returned, signifying that the creation was
+/// successful.
+pub fn init_concordium_project(path: impl AsRef<Path>) -> anyhow::Result<bool> {
+    let path = path.as_ref();
+
+    let absolute_path = if path.is_absolute() {
+        path.to_path_buf()
+    } else {
+        env::current_dir()?.join(path)
+    };
+
+    let result = Command::new("cargo")
+        .arg("generate")
+        .args(&[
+            "--git",
+            "https://github.com/Concordium/concordium-rust-smart-contracts",
+            "--branch",
+            "164-add-default-and-cis2-nft-smart-contract-templates",
```

Comment:
I suppose this should be changed/removed before we merge this PR.

Reply:
yes, the description of this pull request has a note about it:

Once the templates are merged (MR165), we can update the link in cargo-concordium to pull from the main branch rather than MR165.

Reply:
Ah, missed it. Thanks.